compiler/cpp/src/lexer/lexer.cpp

```diff
@@ -61,6 +61,32 @@ Token getTok() {
     // Are we in payload context?
     if (currentContext == PAYLOAD) return consumePayload();
 
+    // Is it a conditional comment identifier?
+    std::string laResult;
+    if (isLookaheadLineCommentChars()) {
+        // Consume LineCommentChars
+        for (int i = 0; i < LineCommentChars.length(); i++) advance();
+        // Update context
+        currentContext = SECTION;
+        return Token(TOK_COM_LINE_IDEN, LineNum, ColNum);
```

Comment:
`LineNum` and `ColNum` should be lower-caps

Reply:
Good catch! Looks like I forgot them when doing the refactoring tasks